scripts/install-aspnet-codegenerator.cmd

```diff
@@ -1,5 +1,5 @@
-set VERSION=9.0.0-dev
-set DEFAULT_NUPKG_PATH=%userprofile%/.nuget/packages
+set VERSION=9.0.1-dev
+set DEFAULT_NUPKG_PATH=C:\Nuget
```

Comment:
This is probably not what we want, right? (I note it's what was already in the branch so it wasn't a change there)

Reply:
on my devbox, the `USERPROFILE` path does not work, I'll just add it to search for both paths. Or check if there is a default nuget path environment variable. Will fix this in the next PR

Reply:
If you mean an actual devbox and not just your dev box, devbox overrides the nuget package location by setting the NUGET_PACKAGES environment variable, which overrides the default `%userprofile%\.nuget\packages` location (for me it points to `q:\.tools\.nuget\packages\`).